vite-plugin-cjs-interop/src/index.ts

```diff
@@ -61,53 +70,119 @@ export function cjsInterop(options: CjsInteropOptions): Plugin {
 			});
 
 			const toBeFixed: any[] = [];
-			const dynamicImportsToBeFixed: any[] = [];
 			const preambles: string[] = [];
+			let hasDynamicImportsToFix = false;
```

Comment:
For simplicity, all fixes are included in toBeFixed.

The dynamicImportsToBeFixed is turned to a hasDynamicImportsToFix boolean to replace the once case where we used it as such (`if dynamicImportsToBeFixed.length`)